ext/afform/admin/afformEntities/Event.php

```diff
@@ -0,0 +1,8 @@
+<?php
+return [
+  'type' => 'primary',
+  'defaults' => "{
+    actions: {create: true, update: false}
```

Comment:
Per @aydun's review I think we should change this default as it's more standard.

```suggestion
    actions: {create: true, update: true}
```